doc/classes/CanvasItem.xml

```diff
@@ -310,16 +310,16 @@
 				# If using this method in a script that redraws constantly, move the
 				# `default_font` declaration to a member variable assigned in `_ready()`
 				# so the Control is only created once.
-				var default_font = Control.new().get_font("font")
-				var default_font_size = Control.new().get_font_size("font_size")
+				var default_font = Control.new().get_theme_font("font")
+				var default_font_size = Control.new().get_theme_font_size("font_size")
```

Comment:
```suggestion
				var default_font = ThemeDB.fallback_font
				var default_font_size = ThemeDB.fallback_font_size
```

Or you can fetch the default theme and get a random value from it. I assume something similar is possible in C#.

Reply:
That'd be much better than constructing a new `Control` every time you want to access them.